src/NHSD.BuyingCatalogue.Solutions.API/ViewModels/Suppliers/AboutSupplierSectionAnswers.cs

```diff
@@ -0,0 +1,24 @@
+using Newtonsoft.Json;
+using NHSD.BuyingCatalogue.Solutions.Contracts.Suppliers;
+
+namespace NHSD.BuyingCatalogue.Solutions.API.ViewModels.Suppliers
+{
+    public sealed class AboutSupplierSectionAnswers
+    {
+        [JsonProperty("description")]
+        public string SupplierDescription { get; }
```

Comment:
Probably worth calling the variables in answers the same as the ones in supplier. E.g. Description